src/content.js

```diff
@@ -116,8 +143,19 @@ module.exports = function (req, res) {
     async.apply(layout, presented)
   ], function (err, result) {
     if (err) {
-      logger.error("Assembling: " + err);
-      res.status(500).send(page500);
+      var code = 404;
+      if(err.type == "NotFound") {
```

Comment:
Tiny nit: In JavaScript you should prefer `===` and `!==` for comparison because it doesn't do the weird, unpredictable, asymmetric implicit type conversions. (It shouldn't matter in this particular case, but still.)